src/sallyport/mod.rs

```diff
@@ -250,23 +250,19 @@ impl<'short, 'a: 'short> Cursor<'a> {
     }
 
     /// Copies data from a slice into the cursor buffer using self.alloc().
```

Comment:
Unrelated to this PR, but should this not be "copies from a cursor buffer into a slice"?

Reply:
:+1: